jiant/scripts/postproc/glue_submission_formatter.py

```diff
@@ -0,0 +1,59 @@
+"""Translate raw prediction files for GLUE tasks into format expected by GLUE leaderboard.
+This script translates raw prediction files for GLUE tasks into the tsv files required
+by the GLUE leaderboard. See https://super.gluebenchmark.com/faq for leaderboard info.
+"""
+import os
+import csv
+import argparse
+import torch
+
+from jiant.tasks import retrieval
+from jiant.tasks.constants import GLUE_TASKS
+
+# this maps the GLUE tasks to the filenames expected by the GLUE benchmark
+# (https://gluebenchmark.com/)
+formatted_pred_output_filenames = {
+    "cola": "CoLA.tsv",
+    "sst": "SST-2.tsv",
+    "mrpc": "MRPC.tsv",
+    "stsb": "STS-B.tsv",
+    "mnli": "MNLI-m.tsv",
+    "mnli_mismatched": "MNLI-mm.tsv",
+    "qnli": "QNLI.tsv",
+    "qqp": "QQP.tsv",
+    "rte": "RTE.tsv",
+    "wnli": "WNLI.tsv",
+    "glue_diagnostics": "AX.tsv",
+}
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Generate output files for GLUE benchmark submission"
+    )
+    parser.add_argument(
+        "--input_base_path",
+        required=True,
+        help="base input path of GLUE tasks (contains task folders)",
+    )
+    parser.add_argument("--output_path", required=True, help="output path for formatted files")
+
+    args = parser.parse_args()
+
+    # for task_name, input_filepath in raw_pred_input_filepaths.items():
+    for task_name in GLUE_TASKS:
+        input_filepath = os.path.join(args.input_base_path, task_name, "test_preds.p")
```

Comment:
Add a README for documenting how to run this? (Can be a later PR)

Reply:
Great idea! I'll add that as a subsequent PR shortly.

Reply:
Great point...will add this is a second PR.